projects/js-packages/analytics-ui/package.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "@automattic/analytics-ui",
+	"version": "0.1.0-alpha",
+	"private": true,
+	"description": "Analytics related UX to be shared across Automattic products.",
+	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/js-packages/analytics-ui/#readme",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Analytics Ui"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/js-packages/analytics-ui"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"exports": {
+		".": "./src/index.jsx"
+	},
+	"scripts": {
```

Comment:
Feels like all that's left is storybook config and build scripts, before there is a full component development env. Worth adding them?